$NetBSD: patch-cvpcb_CMakeLists.txt,v 1.5 2022/01/02 19:10:03 tnn Exp $

Force proper LDFLAGS for MODULE, as cmake doesn't set rpath for these

--- cvpcb/CMakeLists.txt.orig	2021-12-22 13:32:19.000000000 +0000
+++ cvpcb/CMakeLists.txt
@@ -96,6 +96,9 @@ if( MAKE_LINK_MAPS )
         LINK_FLAGS "-Wl,-cref,-Map=_cvpcb.kiface.map" )
 endif()
 
+set_property(TARGET cvpcb_kiface APPEND_STRING
+	PROPERTY LINK_FLAGS " $ENV{LDFLAGS}")
+
 # these 2 binaries are a matched set, keep them together:
 if( APPLE )
     # puts binaries into the *.app bundle while linking
